pkg/operator/staleconditions/remove_stale_conditions.go

```diff
@@ -0,0 +1,116 @@
+package staleconditions
+
+import (
+	"fmt"
+	"time"
+
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
+	"k8s.io/klog"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
+const workQueueKey = "key"
+
+type RemoveStaleConditions struct {
+	conditions []string
+
+	operatorClient v1helpers.StaticPodOperatorClient
+	cachesToSync   []cache.InformerSynced
+
+	eventRecorder events.Recorder
```

Comment:
`eventRecorder` is unused.

Reply:
> `eventRecorder` is unused.

seems like that's a thing to be fixed at some later date.